cmd/security-secrets-setup/README.md

```diff
@@ -1,16 +1,88 @@
 # Testing Instruction for security-secrets-setup
 
-## Build
+## Executable Build
 
-Use Makefile in the base directory of this repository to build the docker image of security-secretes-setup:
+Use Makefile in the base directory of this repository to build the binary executable of security-secrets-setup:
+
+```sh
+make cmd/security-secrets-setup/security-secrets-setup
+```
+
+It should create a binary executable under the directory *cmd/security-secrets-setup/* if successfully built.
+
+## Executable Run [generate] subcommand
+
+To run the **generate** subcommand in linux system, change directory to *cmd/security-secrets-setup/* and then run 
+
+```sh
+sudo -E ./security-secrets-setup generate
+```
+
+Verify that it runs successfully with generating both Vault and Kong's TLS assets under `/run/edgex/secrets/` directory.
```

Comment:
Will also be able to fix some of this when support for `configuration.toml` is added, as /run is really for docker, and it can be in the `res/docker/configuraiton.toml` and the standard `configuration.toml` can just write something to `/tmp/somewhere`

Reply:
yea, will need to modify this doc. again when that feature is in.